cpp/src/experimental/louvain.cuh

```diff
@@ -15,28 +15,20 @@
  */
 #pragma once
 
-#include <thrust/binary_search.h>
+#include <community/dendrogram.cuh>
 
 #include <experimental/graph.hpp>
-
-#include <rmm/thrust_rmm_allocator.h>
-#include <compute_partition.cuh>
-#include <experimental/shuffle.cuh>
-#include <utilities/graph_utils.cuh>
-
-#include <raft/device_atomics.cuh>
-
 #include <experimental/graph_functions.hpp>
+
 #include <patterns/copy_to_adj_matrix_row_col.cuh>
 #include <patterns/copy_v_transform_reduce_in_out_nbr.cuh>
```

Comment:
Do you still need this?

Reply:
The blank line?  Perhaps not.  It prevents clang-format from reordering includes across blocks, I felt like the patterns should be after the graph and functions definitions... but I haven't verified that it's required.

Reply:
No,  #include <patterns/copy_v_transform_reduce_in_out_nbr.cuh>

AFAIK, this was used to compute weight_sums which has been replaced by a graph_view_t class member function.

Reply:
Ah, yes.  You are correct.  I have deleted that.

Reply:
Spoke too soon.  I actually still need it for a different purpose.

https://github.com/rapidsai/cugraph/blob/9e4421e743f7ddbbf64a1eefcfa46e3ef4846c53/cpp/src/experimental/louvain.cuh#L316